scarlet/bbox.py

```diff
@@ -0,0 +1,171 @@
+import autograd.numpy as np
+
+
+class BoundingBox(object):
```

Comment:
Is this class actually used anywhere?

Reply:
Yes, monotonicity uses it to save CPU time and only apply monotonicity to the bounding box. Symmetry uses it as well for the same reason.